### ISSUE TITLE
Same old CUDA error

### ISSUE THREAD

**Issue by massimilianogoi** [LeelaChessZero/lc0#1391]
C:\Users\Massimiliano Goi\Desktop\New folder (3)>lc0.exe
       _
|   _ | |
|_ |_ |_| v0.26.0 built Jul  2 2020
go nodes 100
Found pb network file: C:\Users\Massimiliano Goi\Desktop\New folder (3)/703350.p
b.gz
Creating backend [cudnn-auto]...
Switching to [cudnn]...
CUDA Runtime version: -30076.-56.0
WARNING: CUDA Runtime version mismatch, was compiled with version 10.0.0
Cudnn version: 7.4.2
Latest version of CUDA supported by the driver: 11.0.0
error CUDA error: unknown error (c:\projects\lc0\src\neural\cuda\network_cudnn.c
c:227)

everything is up to date...

**Comment by cn4750** [comment]
You are more likely to get effective help at our Discord's #help channel: [https://discord.gg/pKujYxD](https://discord.gg/pKujYxD)

**Comment by LouisZulli** [comment]
What CUDA Runtime do you think you have installed? Something seems wrong with your installation, since the Runtime major version is interpreted as -30076 and the minor version as -56.

**Comment by borg323** [comment]
Most likely you have a different cuda version installed on your system, which lc0 picks up - if you downloaded `lc0-v0.26.0-windows-gpu-nvidia-cuda-nodll.zip`, try `lc0-v0.26.0-windows-gpu-nvidia-cuda.zip` instead.

**Comment by massimilianogoi** [comment]
> Most likely you have a different cuda version installed on your system, which lc0 picks up - if you downloaded `lc0-v0.26.0-windows-gpu-nvidia-cuda-nodll.zip`, try `lc0-v0.26.0-windows-gpu-nvidia-cuda.zip` instead.

The one I downloaded is https://github.com/LeelaChessZero/lc0/releases/download/v0.26.0/lc0-v0.26.0-windows-gpu-nvidia-cuda.zip

**Comment by massimilianogoi** [comment]
I have installed the Cuda Toolkit 10.0, the latest available for my operating system, Windows 8.1. I got this error:

C:\lc0>lc0.exe
       _
|   _ | |
|_ |_ |_| v0.26.0 built Jul  2 2020
go nodes 100
Found pb network file: C:\lc0/703350.pb.gz
Creating backend [cudnn-auto]...
Switching to [cudnn]...
CUDA Runtime version: 1444159.48.8
WARNING: CUDA Runtime version mismatch, was compiled with version 10.0.0
Cudnn version: 7.4.2
Latest version of CUDA supported by the driver: 11.0.0
error CUDA error: unknown error (c:\projects\lc0\src\neural\cuda\network_cudnn.c
c:227)

In my opinion forcing the users to install Windows 10 is a suicidal technique.

> What CUDA Runtime do you think you have installed? Something seems wrong with your installation, since the Runtime major version is interpreted as -30076 and the minor version as -56.
> 
> Mine looks like this:
> 
> ```
> $ ./lc0
>        _
> |   _ | |
> |_ |_ |_| v0.26.0 built Jul  9 2020
> go nodes 100
> Found pb network file: ./384x30-t60-4300.pb.gz
> Creating backend [cudnn-auto]...
> Switching to [cudnn-fp16]...
> CUDA Runtime version: 11.0.0
> Cudnn version: 7.6.5
> Latest version of CUDA supported by the driver: 11.0.0
> GPU: GeForce RTX 2080 Ti
> GPU memory: 10.7534 Gb
> GPU clock frequency: 1635 MHz
> GPU compute capability: 7.5
> info depth 1 seldepth 2 time 3755 nodes 4 score cp 12 nps 400 tbhits 0 pv d2d4 g8f6
> info depth 2 seldepth 3 time 3761 nodes 11 score cp 10 nps 687 tbhits 0 pv g1f3 d7d5 d2d4
> info depth 3 seldepth 4 time 3766 nodes 37 score cp 11 nps 1761 tbhits 0 pv g1f3 e7e6 c2c4 d7d5
> info depth 3 seldepth 5 time 3776 nodes 104 score cp 12 nps 3250 tbhits 0 pv d2d4 g8f6 c1f4 g7g6 c2c4
> bestmove d2d4 ponder g8f6
> ```
> 
> Error seems to be generated from this line of code:
> 
> ReportCUDAErrors(cudaGetDeviceCount(&total_gpus));

**Comment by Naphthalin** [comment]
Were you able to resolve this?

**Comment by massimilianogoi** [comment]
> Were you able to resolve this?

No. I switched to Windows 10... The compatibility with 8.1 and 7 must be done for the CUDA version in some way because it could get a wider range of users, otherwise it's a loss for the community of lc0.

**Comment by Naphthalin** [comment]
Since Win7 and Win8 aren't supported anymore, this won't be addressed anymore.
